doc/api/esm.md

```diff
@@ -1194,29 +1201,34 @@ export async function resolve(specifier, context, defaultResolve) {
 }
 ```
 
-#### <code>getFormat</code> hook
+#### `getFormat(url, context, defaultGetFormat)`
 
 > Note: The loaders API is being redesigned. This hook may disappear or its
 > signature may change. Do not rely on the API described below.
 
+* `url` {string}
+* `context` {Object}
+* `defaultGetFormat` {Function}
+* Returns: {Object}
+  * `format` {string}
```

Comment:
Same question as above —  is this supposed to be documented as an object wrapped in a promise (compare the JSDoc below)?